internal/peer/common/common.go

```diff
@@ -255,25 +255,29 @@ func configFromEnv(prefix string) (address, override string, clientConfig comm.C
 		secOpts.ServerRootCAs = [][]byte{caPEM}
 	}
 	if secOpts.RequireClientCert {
-		keyPEM, res := ioutil.ReadFile(config.GetPath(prefix + ".tls.clientKey.file"))
-		if res != nil {
-			err = errors.WithMessage(res,
-				fmt.Sprintf("unable to load %s.tls.clientKey.file", prefix))
-			return
-		}
-		secOpts.Key = keyPEM
-		certPEM, res := ioutil.ReadFile(config.GetPath(prefix + ".tls.clientCert.file"))
-		if res != nil {
-			err = errors.WithMessage(res,
-				fmt.Sprintf("unable to load %s.tls.clientCert.file", prefix))
+		secOpts.Key, secOpts.Certificate, err = getClientAuthInfoFromEnv(prefix)
+		if err != nil {
 			return
 		}
-		secOpts.Certificate = certPEM
 	}
 	clientConfig.SecOpts = secOpts
 	return
 }
 
+// getClientAuthInfoFromEnv reads client tls key file and cert file and returns the bytes for the files
+func getClientAuthInfoFromEnv(prefix string) ([]byte, []byte, error) {
+	keyPEM, res := ioutil.ReadFile(config.GetPath(prefix + ".tls.clientKey.file"))
+	if res != nil {
+		return nil, nil, errors.WithMessage(res, fmt.Sprintf("unable to load %s.tls.clientKey.file", prefix))
```

Comment:
Nit (`WithMessagef` didn't exist when a lot of this code was written):
```suggestion
		return nil, nil, errors.WithMessagef(res, "unable to load %s.tls.clientKey.file", prefix)
```